compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -1076,6 +1076,25 @@ pub struct VarDebugInfo<'tcx> {
 // BasicBlock
 
 rustc_index::newtype_index! {
+    /// A node in the MIR [control-flow graph][CFG].
+    ///
+    /// There are no branches (e.g., `if`s, function calls, etc.) within a basic block, which makes
+    /// it easier to do [data-flow analyses] and optimizations. Instead, branches are represented
+    /// as an edge in a graph between basic blocks.
+    ///
+    /// Basic blocks consist of a series of [statements][Statement], ending with a
+    /// [terminator][Terminator]. Basic blocks can have multiple predecessors and successors,
+    /// however there is a MIR pass ([`CriticalCallEdges`]) that removes *critical edges*, which
+    /// are edges that go from a multi-successor node to a multi-predecessor node. This pass is
+    /// needed because some analyses require that there are no critical edges in the CFG.
+    ///
+    /// Read more about basic blocks in the [rustc-dev-guide][guide-mir].
```

Comment:
Should this be in italics?

Reply:
No idea -- which part of the line do you even mean? I'd suggest you follow the usual style of nearby files.

Reply:
I meant the whole line. From this:

> Read more about basic blocks in the rustc-dev-guide.

To this:

> _Read more about basic blocks in the rustc-dev-guide._

That's how it's done in `std::slice`:

![image](https://user-images.githubusercontent.com/37223377/93679939-6bc47e80-fa62-11ea-82b2-2867ebae0f47.png)

But it looks like it's not done that way in the `mir` module:

![image](https://user-images.githubusercontent.com/37223377/93680518-a0d0d100-fa62-11ea-9cc2-db97bc73a29f.png)

Reply:
It's a bikeshed though, so I'll just leave it as it is.

Reply:
`slice` seems like a different situation, this accounts for the fact that the docs for the type and for most of its methods are split across two pages for technical reasons.

So yeah probably just leave it as-is here.